src/main/java/io/confluent/connect/elasticsearch/ConfigCallbackHandler.java

```diff
@@ -194,9 +177,6 @@ private PoolingNHttpClientConnectionManager configureConnectionManager() {
         cm = new PoolingNHttpClientConnectionManager(ioReactor);
       }
 
-      cm.setDefaultMaxPerRoute(config.maxInFlightRequests());
```

Comment:
is this deprecated and what is the impact of removing it?

Reply:
The idea here was to use defaults. Defaults should be more than enough for cloud, but now I'm thinking that there could be on-prem cases where the number of maxInFlightRequests is bigger than that (not likely, but possible). I'll update this to the max + some buffer (setting it to exactly maxInFlightRequests is not a good idea).